app/serializers/v1/billable_metric_serializer.rb

```diff
@@ -10,7 +10,8 @@ def serialize
         description: model.description,
         aggregation_type: model.aggregation_type,
         created_at: model.created_at.iso8601,
-        field_name: model.field_name
+        field_name: model.field_name,
+        group: model.groups_as_tree
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.